### ISSUE TITLE
BUG: Reports input="hidden" fields as critical

### ISSUE THREAD

**Issue by kringo** [janreges/siteone-crawler#70]
After running crawler fixed a critical issue reported for a hidden field with aria-label="somevalue" and later when I ran w3c validator and it error messaged that hidden fields cannot have aria-labels. Is this a bug in crawler?

for eg.

<input type="hidden" name="_csrf" value="somevalue">

is reported as **Missing aria labels** and **critical**

According to w3c validator, input hidden fields are not supposed to have aria-label or aria-hidden

Check [this](https://rocketvalidator.com/html-validation/an-input-element-with-a-type-attribute-whose-value-is-hidden-must-not-have-any-aria-attributes) and [this](https://rocketvalidator.com/html-validation/the-aria-hidden-attribute-must-not-be-specified-on-an-input-element-whose-type-attribute-has-the-value-hidden)
